### PR TITLE
@ #208 | should increase the amount of inotify watchers on Ubuntu

### PR DESCRIPTION
**Pull request by hoatle**
@phuonglm please review

### CODE REVIEW

File: main-cookbooks/teracy-dev/recipes/inotify.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# see https://github.com/teracyhq/dev/issues/208
+if node['platform'] == 'ubuntu'
+  max_user_watches = node['teracy-dev']['inotify']['max_user_watches']
+
+  # should remove the existing line
+  execute 'remove existing fs.inotify.max_user_watches line' do
+      command "sudo sed -i '/fs.inotify.max_user_watches/d' /etc/sysctl.conf"
+  end
+
+  execute 'add new fs.inotify.max_user_watches config' do
```
Comment (hoatle) [inline main-cookbooks/teracy-dev/recipes/inotify.rb]: only if max_user_watches < 1

**Comment by phuonglm** [comment]
need `sudo sysctl -p` after update sysctl.conf

**Comment by hoatle** [comment]
@phuonglm PR updated

**Comment by phuonglm** [comment]
PR ok for me.

**Pull request merged**